DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -73,5 +76,30 @@ public List<CentreUserDetails> GetUserCentres(AdminUser? adminUser, List<Delegat
 
             return availableCentres.OrderByDescending(ac => ac.IsAdmin).ThenBy(ac => ac.CentreName).ToList();
         }
+
+        public bool TryUpdateUserAccountDetails(string password, string signedInEmail, string firstName, string surname, string email)
+        {
+            var (adminUsers, delegateUsers) = GetUsersByEmailAddress(signedInEmail);
```

Comment:
We should never have more than one adminUser per email address as per db constraints. If it's easy to change the implementation of this method, would be good to get it in (I don't think it's used in many places).

Reply:
Agreed, I added the SingleOrDefault call in my usage of the return from this. Will refactor to put that in the Data Service method instead